index.html

```diff
@@ -64,14 +77,8 @@ <h4 class="lectures-title">CC Ignite</h4>
     <section class="speakers">
       <h3 class="speakers-section-title">Featured Speakers</h3>
       <hr class="orange-line">
-      <article class="speaker-container">
-        <img class="speaker-img" src="./images/speakers/1.jpeg" alt="speaker img">
-        <div class="speaker-info">
-          <h4 class="speaker-name">Yochai Benkler</h4>
-          <p class="speaker-about">Product Designer</p>
-          <p class="talk-about">Create the career you're looking for</p>
-        </div>
-      </article>
+      <div class="grid-container">
```

Comment:
- [ ] In this section the text in red should be `italic`

Reply:
done this too !!